README.md

```diff
@@ -71,13 +71,26 @@ For currently supported operations, see [Commands](##Commands) section.
 
 ### From npm
 
+To install globally:
+
+```sh
+npm install --global @ethersphere/swarm-cli
+```
+
+It may require `sudo` if `npm root --global` is not writable by your
+user.
```

Comment:
Or just make writeable the directory for the user in question

Reply:
that's beyond the scope of this doc... should be obvious.

Reply:
indeed, but this `sudo` run also should be obvious then. 
I think this part of the guide only makes confusion in the heads and unintended usage of the library.
if you install the package by root, you will able to reach all commands by root only, which is not documented.